[ARM/Linux] Build failure - undefined reference to `DBG_DebugBreak_End'

CoreCLR build failed for ARM/Linxux with the following message:
```
...
../libcoreclrpal.a(debug.cpp.o): In function `IsInDebugBreak':
/home/parjong/projects/toolchains/coreclr/src/pal/src/debug/debug.cpp:444: undefined reference to `DBG_DebugBreak_End'
clang: error: linker command failed with exit code 1 (use -v to see invocation)
make[3]: *** [src/pal/src/examples/palexmpl] Error 1
```

--- a/src/pal/src/arch/arm/debugbreak.S
+++ b/src/pal/src/arch/arm/debugbreak.S
@@ -10,5 +10,5 @@
 LEAF_ENTRY DBG_DebugBreak, _TEXT
     EMIT_BREAKPOINT
     bx lr
-LEAF_END DBG_DebugBreak, _TEXT
+LEAF_END_MARKED DBG_DebugBreak, _TEXT
 

